test/test_tensorexpr.py

```diff
@@ -1013,6 +1015,17 @@ def foo(*args):
             ref = foo(*values)
             np.testing.assert_allclose(ref.cpu().numpy(), x.cpu().numpy())
 
+            # Test channels-last
+            for _cur_dim in range(4):
```

Comment:
why do we need this for loop? Looks like this _cur_dim is not used inside the loop.

Reply:
Oh. `_cur_dim` is used to set the `_dim` value while `_dim` is global. This loop is just to set the cat dimension. The sematic equivalent is as follows.
```python
for _cur_dim in range(4):
    def foo(*args):
        args_2 = [v + i for i, v in enumerate(args)]
        v = torch.cat(args_2, dim=_cur_dim)
        return v * v
````